kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/BrokerConfigsResourceIntegrationTest.java

```diff
@@ -358,18 +359,21 @@ public void updateAndReset_existingConfig_returnsDefaultUpdatedAndDefaultAgain()
                             .build()))
                 .build());
 
-    Response responseAfterUpdate =
-        request(
-            "/v3/clusters/" + clusterId
-                + "/brokers/" + brokerId
-                + "/configs/compression.type")
-            .accept(MediaType.APPLICATION_JSON)
-            .get();
-    assertEquals(Status.OK.getStatusCode(), responseAfterUpdate.getStatus());
-
-    GetBrokerConfigResponse actualAfterUpdate =
-        responseAfterUpdate.readEntity(GetBrokerConfigResponse.class);
-    assertEquals(expectedAfterUpdate, actualAfterUpdate);
+    testWithRetry(
+        () -> {
+          Response responseAfterUpdate =
```

Comment:
I realise it's according to the style guide, but that indentation looks truly awful.

Reply:
Done.